static/app/components/feedback/list/feedbackList.tsx

```diff
@@ -124,7 +144,11 @@ export default function FeedbackList() {
                   }
                   onRowsRendered={onRowsRendered}
                   overscanRowCount={5}
-                  ref={registerChild}
+                  ref={e => {
+                    // @ts-expect-error
```

Comment:
What's the error?

Reply:
it's "Cannot assign to current because it is a read-only property."

but if i only put `ref={listRef}` then everything is happy. It's the difference between `LegacyRef` and `RefObject`